templates/kubernetes/deploy/ingress.yml

```diff
@@ -10,8 +10,68 @@ metadata:
     ingress.kubernetes.io/ssl-redirect: "true"
     cert-manager.io/cluster-issuer: clusterissuer-letsencrypt-production
     # CORS
-    nginx.ingress.kubernetes.io/enable-cors: "true"
-    nginx.ingress.kubernetes.io/cors-allow-origin: "<% .Name %>.<% index .Params `frontendSubdomain` %>.<% index .Params `domain` %>"
+    ######################################################################################################################
+    # This snippet automatically enables CORS for the referrer of any HTTP request. 
+    # It is here to simplify CORS headaches for Playground apps only and is *NOT* to be used in a real production setting.
+    # Snippet copied from https://qa.lsproc.com/post/access-control-allow-origin-multiple-origin-domains.
+    ######################################################################################################################
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+        add_header 'Access-Control-Allow-Origin' "$http_origin";
+        if ($http_origin ~* (^https?://([^/]+\.)*)) {
+            set $cors "true";
```

Comment:
can we add add the header here instead? or would that cause problems
since whenever `cors`=true we append the header

then we wont need the catch all block, and this line many times

Reply:
I tried a _far_ simplified version of the configuration snippet and it seems to still work. Tested it with Thought Bubbles as well without any issue. See https://github.com/commit-app-playground/backend-template/pull/7/commits/3f2c8d8ce5a2927f54c2b567771f0c8e9df8f9b7